Exercise_1/index.html

```diff
@@ -0,0 +1,148 @@
+                    centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was
+                    popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and
+                    more recently with desktop publishing software like Aldus PageMaker including versions of Lorem
+                    Ipsum.
+
+                    <img src="img/images.png"
+                         alt="Cracow" width="100" height="100" align="right"/>
+
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown
+                    printer
+                    took a galley of type and scrambled it to make a type specimen book. It has survived not only
+                    five
+                    centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It
+                    was
+                    popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages,
+                    and
+                    more recently with desktop publishing software like Aldus PageMaker including versions of Lorem
+                    Ipsum.
+
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown
+                    printer
+                    took a galley of type and scrambled it to make a type specimen book. It has survived not only
+                    five
+                    centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It
+                    was
+                    popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages,
+                    and
+                    more recently with desktop publishing software like Aldus PageMaker including versions of Lorem
+                    Ipsum.
+                    <sup>2</sup>
+                </main>
+                <footer>
+                    <br><br>
+                    <table align="center">
+                        <tr>
+                            <td>
+                                1.Polish prononuciation ......................................................
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>
+                                2. Information source:Wikipedia
+                            </td>
+                        </tr>
+                        </tr>
+                    </table>
+                </footer>
+
+            </article>
+            <table width="100%">
+                <h3>Tonight in Kraków:</h3>
+                <thead align="left">
+                <th>Event</th>
```

Comment:
`<th>` it is like `<td>`, but for the table head. So please wrap them in `<tr>` as well.